spec/units/todo_list_spec.rb

```diff
@@ -8,6 +8,7 @@
   end
 
   it 'read all the items in the todo list' do
-    todolist = ToDoList.new
+    todo_list.add("Shopping")
+    expect(todo_list.read).to eq("Shopping")
```

Comment:
This test might be improved by adding multiple items to the list and checking that the list contains them all. Just protects against a few more bugs (e.g. if it was only reading the last item)